roles/logins/tasks/main.yml

```diff
@@ -1,62 +1,62 @@
 ---
-- name: Create cron job to restart systemd-logind regularly as workaround for known DBus bug.
+- name: 'Create cron job to restart systemd-logind regularly as workaround for known DBus bug.'
```

Comment:
Consistency is also nice.

Reply:
Personally, I'd still prefer you'd tweak your syntax highlighting instead of introducing a "syntax rule" that is unique for one the projects I work in.

Reply:
I would not consider this a strength of yaml, but rather a design flaw instead. You do not need to quote a string unless  ... and then there is a long list of exceptions. Simply quoting everything is a lot easier than remembering all the exceptions. I'll remove the quotes when string values contain only [a-zA-Z_], but I'll keep them when anything else is involved and when the string value is part of a list and one of the other list items must be quoted.